.github/workflows/ci.yml

```diff
@@ -143,10 +150,13 @@ jobs:
           kubectl get pods
       - name: Run Flink e2e tests
         run: |
+          sed -i "s/image: flink:.*/image: ${{ matrix.versions.image }}/" e2e-tests/data/*.yaml
+          sed -i "s/flinkVersion: .*/flinkVersion: ${{ matrix.versions.flinkVersion }}/" e2e-tests/data/*.yaml
           ls e2e-tests/test_*.sh | while read script_test;do \
```

Comment:
I would like to add a `git diff HEAD` here to verify that the `sed` is executed successfully.